include/old/test.h

```diff
@@ -44,6 +44,9 @@
 #include <string.h>
 #include <stdlib.h>
 #include <stdint.h>
+#ifndef __GLIBC__
+#include <bits/reg.h>
+#endif
 
```

Comment:
`LTP_USE_64_ABI` has a specific use for selecting syscall ABIs.  for the other tests, we should probably set up some autoconf tests to look at specific types or sizes that make sense.